src/main/interface/cli.c

```diff
@@ -2264,6 +2264,24 @@ static void cliName(char *cmdline)
     printName(DUMP_MASTER, pilotConfig());
 }
 
+static void printDisplayName(uint8_t dumpMask, const pilotConfig_t *pilotConfig)
```

Comment:
This (and the code in `cms_menu_osd.c` should be made conditional to `USE_OSD` - the display name will only be used in OSD, so there is no point including this code on targets that don't have OSD.